conda_env/cli/main_create.py

```diff
@@ -22,6 +23,7 @@
     conda env create -n=foo
     conda env create -f=/path/to/environment.yml
     conda env create --name=foo --file=environment.yml
+    conda env create --name=foo --binstar=username/foo
```

Comment:
I agree with @tswicegood it would be great to have this as:
```
conda env create --name=wakari malev/wakari
# or
conda env create malev/wakari
```
But I'm not sure where to have this discussion